<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userForm</class>
 <widget class="QWidget" name="userForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="tableTitle">
         <property name="text">
          <string>FAIRIES I HAVE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="fairy_tableWidget"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>user</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="username_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>textlabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="upgradeBattle_pushButton">
           <property name="text">
            <string>Upgrade Battle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="winLose_pushButton">
           <property name="text">
            <string>Win-lose Battle</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="lookUpOnlineUsers_pushButton">
           <property name="text">
            <string>Look up online users</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="lookUpAllUsers_pushButton">
           <property name="text">
            <string>Look up ALL users</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="signOut_pushButton">
           <property name="text">
            <string>Sign out</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
